lib/stb_image.ex

```diff
@@ -32,18 +34,22 @@ defmodule StbImage do
 
   ## Example
 
-      {:ok, img, shape, type, channels} = StbImage.from_file("/path/to/image")
-      {h, w, c} = shape
+      img = StbImage.from_file("/path/to/image")
+      {h, w, c} = img.shape
+      data = img.data
 
       # If you know the image is a 4-channel image and auto-detection failed
-      {:ok, img, shape, type, channels} = StbImage.from_file("/path/to/image", channels: 4)
-      {h, w, c} = shape
+      stb_image_struct = StbImage.from_file("/path/to/image", channels: 4)
```

Comment:
```suggestion
      img = StbImage.from_file("/path/to/image", channels: 4)
```

or `image`. Please update the other places too :)

Reply:
Ok